pkg/types/azure/validation/platform.go

```diff
@@ -0,0 +1,41 @@
+package validation
+
+import (
+	"sort"
+
+	"github.com/openshift/installer/pkg/types/azure"
+	"k8s.io/apimachinery/pkg/util/validation/field"
+)
+
+var (
+	// Regions is a map of the known azure regions. The key of the map is
+	// the short name of the region. The value of the map is the long
+	// name of the region.
```

Comment:
This list is unecessary if we query azure for the available regions (done here : https://github.com/openshift/installer/blob/a95ce4126732191c1e43503b03ee4b3eee24d521/pkg/asset/installconfig/azure/azure.go#L77), unless you want finer grain control over where openshift can be deployed.

Reply:
removed static region list